model/job/scheduler.go

```diff
@@ -306,5 +307,7 @@ func GetTriggerState(db prefixer.Prefixer, triggerID string) (*TriggerState, err
 	return &state, nil
 }
 
-var _ couchdb.Doc = &TriggerInfos{}
-var _ permission.Fetcher = &TriggerInfos{}
+var (
+	_ couchdb.Doc        = &TriggerInfos{}
+	_ permission.Fetcher = &TriggerInfos{}
+)
```

Comment:
I would prefer to keep the short version (2 lines) instead of the longer version (4 lines) here